resilience4j-metrics/src/test/java/io/github/resilience4j/metrics/TimerTest.java

```diff
@@ -158,9 +160,12 @@ public void shouldExecuteCompletionStageSupplier() throws Throwable {
 
         assertThat(value).isEqualTo("Hello world");
 
-        assertThat(timer.getMetrics().getNumberOfTotalCalls()).isEqualTo(1);
-        assertThat(timer.getMetrics().getNumberOfSuccessfulCalls()).isEqualTo(1);
-        assertThat(timer.getMetrics().getNumberOfFailedCalls()).isEqualTo(0);
+        await().atMost(1, SECONDS)
+                .until(() -> {
+                    assertThat(timer.getMetrics().getNumberOfTotalCalls()).isEqualTo(1);
+                    assertThat(timer.getMetrics().getNumberOfSuccessfulCalls()).isEqualTo(1);
+                    assertThat(timer.getMetrics().getNumberOfFailedCalls()).isEqualTo(0);
+                });
```

Comment:
This change is unrelated to this PR but my build was red because of this flaky test.